ansible/roles/prometheus-server/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+# Prometheus
+prometheus_version: 2.20.1
+prometheus_tar_gz: "prometheus-{{ prometheus_version }}.linux-amd64.tar.gz"
+prometheus_tar_gz_url: "https://github.com/prometheus/prometheus/releases/download/v{{ prometheus_version }}/{{ prometheus_tar_gz }}"
+
+prometheus_root_dir: /opt/prometheus
+prometheus_config_dir: /etc/prometheus
+prometheus_server_dir: "{{ prometheus_root_dir }}/prometheus-server"
+prometheus_dist_dir: "{{ prometheus_root_dir }}/dist"
+prometheus_data_dir: /var/opt/prometheus/data
+
+prometheus_user: "prometheus"
+prometheus_group: "prometheus"
+
+# Volume mounts
+volume_mounts:
+  - device: /dev/xvdb
+    file_system_type: ext4
+    logical_volume: lv_prometheus_data
+    mode: "0755"
+    mount_point: "{{ prometheus_data_dir }}"
+    group: "{{ prometheus_group }}"
+    owner: "{{ prometheus_user }}"
+    volume_group: vg.prometheus
+    options:
+      - defaults
```

Comment:
I would suggest renaming the volume group here to the generic term `data` that we're adopting in other builds, and simplifying the logical volume name to `prometheus` which is distinct within the group itself.

Reply:
Done. Volume group and logical volume both renamed.